deploy/kubernetes/helm/values.yaml.template

```diff
@@ -53,18 +54,18 @@ uploader:
 packing: RoundRobin # ResourceCompliantRR, FirstFitDecreasing
 
 # Number of replicas for storage bookies, memory and storage requirements
-bookieReplicas: 1
+bookieReplicas: 3
 bookieCpuMin: 100m
 bookieCpuMax: 250m
-bookieHeapMemory: 256M
-bookieDirectMemory: 128M
+bookieHeapMemory: 512M
+bookieDirectMemory: 256M
 bookieReadCacheSize: 32
 bookieWriteCacheSize: 32
-bookieJournalCapacity: 1G
-bookieStorageCapacity: 3G
+bookieJournalCapacity: 10G
```

Comment:
10G and 30G look quite big and could be a limitation for some people. Are they really necessary?

Reply:
we are finding using the shade plugin causes large jars and deploying a lot of jars the old configs were problematic.  This seems to be a happy spot.  I can reduce it.